node/src/main.rs

```diff
@@ -150,6 +150,20 @@ async fn main() {
                        which will share the load with the primary server",
                 ),
         )
+        .arg(
+            Arg::with_name("postgres-host-weigths")
+                .multiple(true)
```

Comment:
Is this really a flag that requires `multiple(true)`? It's just a single CSV value.

Reply:
I just followed what `postgres-secondary-hosts` does; not sure if you would want to specify the weights similarly